core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -143,6 +143,10 @@ public static void checkActualAndReferenceFiles() {
     sql("select interval mgr hour as h from emp").ok();
   }
 
+  @Test void testIntervalSecondNoFractionalPart() {
```

Comment:
it is customary to add a comment with the JIRA issue that is being solved.
Please follow the other examples in this file that start with "Test case for".
Note that everything is important, including the last period.

Reply:
Thanks, fixed.